examples/kubernetes-labels/kubernetes.json

```diff
@@ -0,0 +1,40 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorVersion": "1.6.2"

```

Comment:
ACS now uses v1.6.6 as the default Kubernetes version. Can you update your template?